modules/sdk-coin-avaxp/src/lib/exportTxBuilder.ts

```diff
@@ -0,0 +1,128 @@
+import { BaseCoin as CoinConfig } from '@bitgo/statics';
+import { BuildTransactionError, NotSupported, TransactionType } from '@bitgo/sdk-core';
+import { TransactionBuilder } from './transactionBuilder';
+import {
+  BaseTx,
+  ExportTx,
+  PlatformVMConstants,
+  SECPTransferOutput,
+  TransferableOutput,
+  Tx,
+  UnsignedTx,
+} from 'avalanche/dist/apis/platformvm';
+import utils from './utils';
+import { BN, Buffer as BufferAvax } from 'avalanche';
+import { AmountOutput } from 'avalanche/dist/apis/evm/outputs';
+
+export class ExportTxBuilder extends TransactionBuilder {
+  private _amount: BN;
+  private _targetChainId: BufferAvax;
+
+  constructor(_coinConfig: Readonly<CoinConfig>) {
+    super(_coinConfig);
+    this.transaction._fee = new BN(this.transaction._network.txFee);
+  }
+
+  protected get transactionType(): TransactionType {
+    return TransactionType.Export;
+  }
+
+  /**
+   *
+   * @param value
+   */
+  targetChainId(chainId: string | Buffer): this {
+    const newTargetChainId = typeof chainId === 'string' ? utils.cb58Decode(chainId) : BufferAvax.from(chainId);
+    this.validateChainId(newTargetChainId);
+    this._targetChainId = newTargetChainId;
+    return this;
+  }
+
+  /**
+   *
+   * @param value
+   */
+  amount(value: BN | string): this {
+    const valueBN = BN.isBN(value) ? value : new BN(value);
+    this.validateAmount(valueBN);
+    this._amount = valueBN;
+    return this;
+  }
+
+  /**
+   *
+   * @param amount
+   */
+  validateAmount(amount: BN): void {
+    if (amount.lten(0)) {
+      throw new BuildTransactionError('Amount must be greater than 0');
+    }
+  }
+
+  /**
+   *
+   * @param chainID
+   */
+  private validateChainId(chainID: BufferAvax) {
```

Comment:
Is there a reason to make this method private?
This method and `validateAmount` could also be static
```suggestion
  static validateChainId(chainID: BufferAvax): void {
```